pwndbg/dbg/lldb/repl/readline.py

```diff
@@ -9,11 +9,19 @@
 import contextlib
 import functools
 import os.path
+import sys
 from typing import Callable
 from typing import ParamSpec
 from typing import TypeVar
 
-import gnureadline as readline
+if sys.platform == "win32":
```

Comment:
Can we do `if sys.platform != "win32": gnureadline; else: windows`?

Reply:
fixed